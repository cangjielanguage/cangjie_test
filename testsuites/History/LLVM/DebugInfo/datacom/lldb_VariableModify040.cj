// DEPENDENCE: ../cjcmd.py ../DatacomTest.py lldb_VariableModify040_expected.txt ../basecase/024_debug_variable.cj
// EXEC: cjc -g 024_debug_variable.cj
// EXEC: python3 DatacomTest.py lldb_VariableModify040_expected.txt



