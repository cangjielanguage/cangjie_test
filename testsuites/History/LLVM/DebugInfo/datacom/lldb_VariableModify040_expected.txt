lldb,lldb 024_debug_variable,024_debug_variable
b,b 44,024_debug_variable.cj:44
r,r,-> 44
cj_set,cj_set var_float32 = "aaa",error
n,n,-> 47
cj_plocals,cj_plocals,var_float32 = 1
cj_plocals,cj_plocals,f_float32 = 9